core/src/main/java/org/openstack4j/openstack/magnum/MagnumBay.java

```diff
@@ -7,172 +7,273 @@
 import org.openstack4j.openstack.common.GenericLink;
 import org.openstack4j.openstack.common.ListResult;
 
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import com.fasterxml.jackson.annotation.JsonProperty;
-import com.google.common.base.MoreObjects;
```

Comment:
Since Objects.ToStringHelper is depreated, I have changed it to MoreObjects.ToStringHelper.